source/common/event/timer_impl.h

```diff
@@ -22,6 +22,9 @@ class TimerImpl : public Timer, ImplBase {
   void enableTimer(const std::chrono::milliseconds& d) override;
   bool enabled() override;
 
+  // Public for testing.
+  void millisecondsToTimeval(timeval* tv, const std::chrono::milliseconds& d);
```

Comment:
Suggest putting in a utility class, e.g. `TimerUtils` and making `static`

Reply:
Ok. I was originally aiming to do "minimal" changes but a separate class makes sense.